cloudwatch_metrics_common/include/cloudwatch_metrics_common/utils/metric_object.h

```diff
@@ -84,11 +121,27 @@ static MetricDatum metricObjectToDatum(const MetricObject &metrics, const int64_
     datum.WithUnit(Aws::CloudWatch::Model::StandardUnitMapper::GetStandardUnitForName(unit_name));
   }
 
+  if (!metrics.statistic_values.empty()) {
```

Comment:
The comment on line 102 is no longer valid

Reply:
Removed.